resources/guis.yml

```diff
@@ -73,7 +73,7 @@ guis:
   platforms:
   - Windows
   - Mac
-  price: " $69/user (Free 30 day trial)"
+  price: "Free for students and educators / $69/user annually / $99/user annually (Free 30 day trial)"
```

Comment:
same as 1st comment. here in particular, the price is 69$. it happens there's a promotion to students sure, but many tools do that as well

Reply:
Agree here too, will update